src/amp-story-player/amp-story-player-impl.js

```diff
@@ -673,24 +680,56 @@ export class AmpStoryPlayer {
     };
   }
 
-  /**
-   * @public
-   */
-  layoutCallback() {
-    if (this.isLaidOut_) {
-      return;
-    }
-
+  /** @private */
+  prerenderStories_() {
     for (let idx = 0; idx < this.stories_.length && idx < MAX_IFRAMES; idx++) {
       const story = this.stories_[idx];
       const {iframeIdx} = story;
       const iframe = this.iframes_[iframeIdx];
-      this.layoutIframe_(
+
+      this.updateIframeSrc_(
         story,
         iframe,
-        idx === 0 ? VisibilityState.VISIBLE : VisibilityState.PRERENDER
+        VisibilityState.PRERENDER,
+        idx === 0 /** isFirstStory */
```

Comment:
I think this parameter should be named something that can be re-used in the future for other use cases. If I understand correctly, it means "please wait for this one to be loaded before loading anything else", which we can use to optimize the `show()` API later